extensions/azurecore/src/account-provider/providerSettings.ts

```diff
@@ -84,7 +84,7 @@ const publicAzureSettings: ProviderSettings = {
 			},
 			azureKustoResource: {
 				id: SettingIds.kusto,
-				endpoint: 'https://api.kusto.io',
+				endpoint: 'https://kusto.kusto.windows.net',
```

Comment:
Do you know why this changed?

And is `kusto.kusto` correct? (double kusto)

Reply:
Yes this is correct. I got the correct information from Vladick Branevich in Kusto team